### ISSUE TITLE
Element -> Expand Stroke results in "Internal Error: Some fragments did not join" 

### ISSUE THREAD

**Issue by ctrlcctrlv** [fontforge/fontforge#2406]
Just trying to create a simple number sign...

![numsign1](https://cloud.githubusercontent.com/assets/838783/8507873/043765b8-2289-11e5-9deb-32ebcff82cc1.png)
![numsign3](https://cloud.githubusercontent.com/assets/838783/8507871/03f68a3e-2289-11e5-9d05-1152aaa042d7.png)
![numsign4](https://cloud.githubusercontent.com/assets/838783/8507883/481e2a0a-2289-11e5-8e1e-33d8e23d421d.png)
![numsign2](https://cloud.githubusercontent.com/assets/838783/8507872/03fd094a-2289-11e5-9596-e2a5d47ebb6e.png)

Please fix this, I have no idea what it is about this very simple shape that is causing problems ;_;
```
StartChar: numbersign
Encoding: 35 35 87
Width: 518
VWidth: 0
Flags: HW
LayerCount: 2
Fore
SplineSet
469 311 m 29
 28 311 l 1053
499 561 m 29
 58 561 l 1053
420 798 m 29
 312 0 l 29
 420 798 l 29
200 798 m 29
 92 0 l 29
 200 798 l 29
EndSplineSet
EndChar
```

**Comment by ctrlcctrlv** [comment]
Perhaps I am not even going about this the right way?

None of the tutorials I watch seem to use `Expand Stroke`, but I feel that this function is best for the font I am designing, which is a cursive font for education, not display. I found that the existing fonts for this lack ligatures, so some joins are very silly looking (e.g. `or`, `er`, `os`, three ligatures I have already made)...

![sample2](https://cloud.githubusercontent.com/assets/838783/8510550/36d4eb70-2320-11e5-8681-8ae254a9b2d7.png)

However, most tutorials are focusing on stroke direction, etc, things I just ignored...
